reference/docs-conceptual/developer/format/propertycountfortable-element-format.md

```diff
@@ -1,17 +1,44 @@
 ---
-ms.date: 09/12/2016
+ms.date: 08/17/2021
 ms.topic: reference
 title: PropertyCountForTable Element (Format)
 description: PropertyCountForTable Element (Format)
 ---
 # PropertyCountForTable Element (Format)
 
-Insert introduction here.
+Optional element. Specifies the minimum number of properties that an object must have to display the
+object in a table view.
 
-## Section Heading
+ConfigurationSettings Element (Format)
```

Comment:
I have a question which will affect most of these docs. Since `ConfigurationSettings` isn't an element, it's a complext type [according to the schema](https://github.com/PowerShell/PowerShell/blob/7d53fbbdf7032c1f293da21108bd287cbe6e00db/src/Schemas/Format.xsd#L32), should this be complex type instead of element?

tag: @sdwheeler

Reply:
The correct element name is `DefaultSettings`.